python/sqlflow_submitter/tensorflow/evaluate.py

```diff
@@ -0,0 +1,202 @@
+# Copyright 2020 The SQLFlow Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import copy
+import functools
+import glob
+import json
+import os
+import sys
+import types
+
+import numpy as np
+import tensorflow as tf
+from sqlflow_submitter.db import (buffered_db_writer, connect_with_data_source,
+                                  db_generator, pai_maxcompute_db_generator,
+                                  parseMaxComputeDSN)
+
+from . import metrics
+from .get_tf_version import tf_is_version2
+from .input_fn import get_dataset_fn
+from .pai_distributed import define_tf_flags, set_oss_environs
+from .set_log_level import set_log_level
+from .train_estimator import estimator_train_and_save
+from .train_keras import keras_train_and_save
```

Comment:
It looks like several packages imported but not used, such as: `functools, glob, json... keras_train_and_save `